Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -134,6 +139,7 @@ public async Task AddAccountToCacheFromAuthorizationCode(AuthorizationCodeReceiv
             }
             catch (MsalException ex)
             {
+                // brentsch - why do we have this statement here and not writting to a log?
```

Comment:
because we have not thought of it.

Reply:
@jmprieur we could use the ILogger service.